lib/cisco_node_utils/bgp_af.rb

```diff
@@ -159,6 +164,34 @@ def default_next_hop_route_map
       config_get_default('bgp_af', 'next_hop_route_map')
     end
 
+    def next_hop_route_policy
+      return nil if platform == :ios_xr && @vrf != 'default'
+      config_get('bgp_af', 'next_hop_route_policy', @get_args)
+    end
+
+    def next_hop_route_policy=(route_policy)
+      fail Cisco::UnsupportedError.new('bgp_af', 'nexthop route-policy', 'set',
```

Comment:
Can you move this code into a local method caled `check_supported` or something along those lines?  That way it can be extended for future cases easily and avoid the repetition.